PGIF.py

```diff
@@ -56,8 +56,8 @@ def setup(self):
             except:
                 print('ERROR: System could not read GIF Image(s)')
             else:
-                image = pygame.transform.scale(image, self.scale_to)
+                image = pygame.transform.smoothscale(image, self.scale_to)
                 self.canvas.blit(image, self.location)
 
         if self.update:
-            pygame.display.update()
+            pygame.display.flip()

```

Comment:
As stated in a documentation, pygame.display.update() only updates a given portion. I could have used flip instead of update, however, I thought it is better idea to use update because in that way user can update a portion of the screen.